js-green-licenses.json

```diff
@@ -17,19 +17,25 @@
 	],
 	"packageWhitelist": [
 		"@eventespresso/react-exit-modal-typeform",
-		"js-green-licenses",
 		"@typeform/embed",
-		"scrollbar-width",
-		"webfontloader",
-		"validate-npm-package-license",
-		"spdx-correct",
-		"domutils",
-		"domelementtype",
-		"nomnom",
+		"@svgr/webpack",
+		"caniuse-lite",
```

Comment:
https://github.com/ben-eb/caniuse-lite/blob/master/LICENSE

Reply:
and https://creativecommons.org/2015/10/08/cc-by-sa-4-0-now-one-way-compatible-with-gplv3/